features/handled_errors.feature

```diff
@@ -1,17 +1,20 @@
 Feature: Handled Errors and Exceptions
 
-Scenario: Override errorClass and message from a notifyError() callback and discard lines from stack
+Scenario: Override errorClass and message from a notifyError() callback, customize report
```

Comment:
proposing adding the comment to the scenario about metadata, please adjust wording as necessary.